articles/Third Party Tools and Libraries/enable-math.md

```diff
@@ -9,21 +9,32 @@ contributors:
     - 'Jérôme Bogaerts'
     - 'Cyril Hazotte'
     - 'Antoine Robin'
+    - 'Jean-Sébastien Conan'
 tags:
-    - Wiki
+    - 'Frontend'
+    - 'Library'
+    - 'JavaScript'
+    - 'Math'
+    - 'MathJax'
+    - 'MathML'
+    - 'LaTex'
+    - 'ThirdParty'
 -->
 
 # Enable math expression in items
 
-![](../resources/badge-square.png)
+![logo](../resources/third-party/badge-square.png)
+
+> MathJAX is a library for rendering Math expressions within a web page.
+> It supports both MathML and LaTex expressions.
```

Comment:
The combination title -> synopsis as one line -> logo is well tested, this order not so much. I'm pretty curious to see if this passes my parser with the expected result. We'll see :-)